src/components/15Years/15YearsPartners.vue

```diff
@@ -462,51 +319,51 @@ export default {
 	}
 
 	&__total-impact-headline {
-		max-width: rem-calc(228);
 		color: $offwhite;
 		letter-spacing: -0.15rem;
-		line-height: 3.5rem;
+		line-height: 1.15em;
 		margin-bottom: 1.5rem;
 	}
 
 	&__partner {
+		padding: rem-calc(12);
 		height: 100%;
 
 		p {
+			font-size: rem-calc(14);
```

Comment:
To help stay consistent, font sizes for p, h2, h3, etc. are already set in 15YearsStyles.vue, so don't need to redo here unless there's an override needed.